exporter/instanaexporter/README.md

```diff
@@ -0,0 +1,73 @@
+# Instana Exporter
+
+| Status                   |                  |
+| ------------------------ |------------------|
+| Stability                | [in-development]  |
```

Comment:
Not for this PR, but the table deserves properly formatting :-)

Reply:
Its is unaligned here in the comparison but it is aligned when the markdown is rendered.